app/api/business/brief/user_status.py

```diff
@@ -78,18 +89,49 @@ def has_been_assessed_for_brief(self):
         return False
 
     def can_respond(self):
-        if (self.user_role == 'supplier' and
-            ((self.brief.data.get('openTo', '') == 'all' and self.is_assessed_in_any_category()) or
-             (self.brief.data.get('openTo', '') == 'category' and self.is_assessed_for_category()) or
-             str(self.supplier_code) in self.invited_sellers.keys())):
+        if (
+            self.user_role == 'supplier' and (
+                (
+                    self.brief.lot.slug == 'specialist' and
+                    self.brief.data.get('openTo', '') == 'all'
```

Comment:
Is this a valid path?  Looks like buyers are only prompted to select a category.

Reply:
add is invited to solve this issue